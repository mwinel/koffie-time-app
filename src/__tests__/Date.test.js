import React from "react";
import { render, unmountComponentAtNode } from "react-dom";
import { act } from "react-dom/test-utils";
import pretty from "pretty";
import CreateDate from "../components/Date";

let container = null;

beforeEach(() => {
  // setup a DOM element as a render target
  container = document.createElement("div");
  document.body.appendChild(container);
});

afterEach(() => {
  // cleanup on exiting
  unmountComponentAtNode(container);
  container.remove();
  container = null;
});

it("renders date component", () => {
  act(() => {
    render(<CreateDate monthName="Dec" />, container);
  });
  expect(container.innerHTML).toBeDefined();
  expect(pretty(container.innerHTML)).toMatchSnapshot();
});
