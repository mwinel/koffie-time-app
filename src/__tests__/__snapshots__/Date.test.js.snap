// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders date component 1`] = `"<small>undefined NaN </small>"`;
